README.md

```diff
@@ -4,12 +4,29 @@
 
 Crowdin IOS SDK delivers all new translations from Crowdin project to the application immediately. So there is no need to update this application via App Store to get the new version with the localization.
 
+The SDK provides:
+
+* Over-The-Air Content Delivery – the localized files can be sent to the application from the project whenever needed
+* Real-time Preview – all the translations that are done via Editor can be shown in the application in real-time
+* Screenshots – all screenshots made in the application may be automatically sent to your Crowdin project with tagged source strings
+
+
+[![GitHub issues](https://img.shields.io/github/issues/crowdin/mobile-sdk-ios)](https://github.com/crowdin/mobile-sdk-ios/issues)
+[![GitHub commit activity](https://img.shields.io/github/commit-activity/m/crowdin/mobile-sdk-ios)](https://github.com/crowdin/mobile-sdk-ios/graphs/commit-activity)
+[![GitHub last commit](https://img.shields.io/github/last-commit/crowdin/mobile-sdk-ios)](https://github.com/crowdin/mobile-sdk-ios/commits/master)
+[![GitHub contributors](https://img.shields.io/github/contributors/crowdin/mobile-sdk-ios)](https://github.com/crowdin/mobile-sdk-ios/graphs/contributors)
+[![GitHub](https://img.shields.io/github/license/crowdin/mobile-sdk-ios)](https://github.com/crowdin/mobile-sdk-ios/blob/master/LICENSE)
+
 
 ## Table of Contents
 * [Requirements](#requirements)
 * [Dependencies](#dependencies)
 * [Installation](#installation)
-* [Quick Start](#quick-start)
+* [Setup](#setup)
+* [Advanced Features](#advanced-features)
+  * [Real-time Preview](#real-time-preview)
+  * [Screenshots](#screenshots)
+  * [Force Update Button](#force-update-button)
```

Comment:
I'm not sure we need "Button" word. Maybe it will be better to name it just "Force Update"?